src/styles/designTokens.js

```diff
@@ -7,6 +7,7 @@ export const colorTurquoiseDimmed = '#008484';
 export const colorRed = '#e0151d';
 export const colorLightRed = '#fadcdd';
 export const colorLightGrey = '#b0b0b0';
+export const colorLightBlue = '#00afd3';
```

Comment:
For now let's not add more colors to the mix - **colorTurquoise** should be ok for this banner.

Reply:
Ok, wasn't sure about it, I was trying to keep the design consistent with the live site. Amended as suggested, will be part of updated PR.